Sources/Array+Remove.swift

```diff
@@ -35,13 +36,13 @@ extension Array {
 
      - parameter object: The object to be removed (if found)
      */
-    mutating func removeObject<T: Equatable>(_ object: T) {
+    public mutating func removeObject<Element: Equatable>(_ object: Element) {
 
         var foundIndex: Int?
```

Comment:
Interesting, not sure those methods existed back in Swift 1.0 but will take a look in future (maybe we can raise an issue for it)